packages/ts-migrate-plugins/src/plugins/ts-ignore.ts

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable no-use-before-define, @typescript-eslint/no-use-before-define */
 import ts from 'typescript';
 import tsp from 'typescript/lib/protocol';
-import diagnosticMessages from 'typescript/lib/diagnosticMessages.generated.json';
```

Comment:
4.0 no longer contains this file. But the `diagnosticMessagesByCode` variable doesn't seem to be used anyway, so I removed all of it.